public/style/_variables.scss

```diff
@@ -0,0 +1,43 @@
+// Hex instead of RGBA for ie8 support
+$gray: #4e554e; // 7.2 contrast with background of $gray-lighter (#f8f8f8)
+$gray-lighter: #f8f8f8;
+$blue: #2b598e;
+$blue-darker: darken($blue, 10%);
+$blue-light: #77b5ed;
+$purple: rebeccapurple;
+// although I really want to use rebeccapurple
+$purple-fallback: #639;
+// Having trouble getting scss to output 'rebeccapurple' instead of #639.
+// #639 has better support
+// #3D00B5 is a possible, higher contrast alternative
+$purple-high-contrast: #3d00b5;
+
+// High contrast colors
+$black: #000;
```

Comment:
More just a good practice thing than anything. My linter complains if there are any directly referenced colors in the scss files